draining_validation/hooks/install.yml

```diff
@@ -2,8 +2,9 @@
 - name: "Perform node draining process"
   vars:
     trex_app_cr_name: "{{ trex_app_cr_name_to_check|default('trex-app-new') }}"
```

Comment:
I suppose you will transform at some point the variables to follow the collections' naming convention (`ecd_trex_app_cr_name`), same in other cases where this is still present.